### DISCUSSION TITLE
Ability to resize gui panels

### DISCUSSION THREAD

**Discussion by fbd-ss** [louislam/dockge#551] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

The ability to resize / hide / dock the 3 "panels/views" on the gui, instead % based scaling for the views.  

### ✔️ Solution

Make each view resizable / hideable.  The list on the left has no reason to scale outside of its contents.  Usually I am only interested in either the logs or the compose viewer, but never both.  Have a toggle to switch between the 2 views

**Comment by mahmoudhamadeh** [comment]
I agree, when troubleshooting a container sometimes I run both Dockge and the service in split screens which makes it hard to read the Terminal output on Dockge, text wrap sometimes trim the text in terminal and I have to expand Dockge window to full screen to read the terminal.

**Comment by jammsen** [comment]
Could be as easy as to add only 2 tabs call "Logs/Terminal" and "docker-compose.yml".

**Comment by tismofied** [comment]
[Similar request in discussions and a temporary workaround](https://github.com/louislam/dockge/discussions/31)
